benches/src/db_lookup_times_utils/matrix.rs

```diff
@@ -0,0 +1,26 @@
+use fuel_core_types::fuel_types::BlockHeight;
+
+pub const BLOCK_COUNT_MATRIX: [u32; 1] = [4];
+pub const TX_COUNT_MATRIX: [u32; 1] = [1];
```

Comment:
Maybe we want to have bigger values here=)

Reply:
addressed in [b196d9d](https://github.com/FuelLabs/fuel-core/pull/2159/commits/b196d9d796b22a068c86bf4b9906a8c7e883ede8)